### ISSUE TITLE
Groups names are not being updated

### ISSUE THREAD

**Issue by osbornk** [jelhub/scimgateway#95]
```
2023-06-20T22:10:07.909 plugin-fulcrum.ts info: plugin-fulcrum.ts handling "getGroups" getObj={"attribute":"id","operator":"eq","value":"fc0a8962-65c4-4402-aae5-bd2fcbe19f40"} attributes=                                                                │
│ 2023-06-20T22:10:08.153 plugin-fulcrum.ts info: plugin-fulcrum.ts handling "modifyGroup" id=fc0a8962-65c4-4402-aae5-bd2fcbe19f40 attrObj={"members":[]}                                                                                                    │
│ 2023-06-20T22:10:08.301 plugin-fulcrum.ts info: plugin-fulcrum.ts handling "getGroups" getObj={"attribute":"id","operator":"eq","value":"fc0a8962-65c4-4402-aae5-bd2fcbe19f40"} attributes=                                                                │
│ 2023-06-20T22:10:08.729 plugin-fulcrum.ts info: scimgateway[plugin-fulcrum.ts] 821ms 34.199.123.185 token PUT http://kosborn.thefulcrum.team/scim/Groups/fc0a8962-65c4-4402-aae5-bd2fcbe19f40 Inbound = {"schemas":["urn:ietf:params:scim:schemas:core:2.0 │
│ :Group"],"id":"fc0a8962-65c4-4402-aae5-bd2fcbe19f40","displayName":"KOTestGroup Test","members":[{"value":"e07986cd-6ce9-47f0-8ba7-c7d316311653","display":null},{"value":"6c320c45-0033-4d10-bdda-3c711d720025","display":null}]} Outbound = {"statusCode │
│ ":200,"statusMessage":"OK","body":{"schemas":["urn:ietf:params:scim:schemas:core:2.0:Group"],"id":"fc0a8962-65c4-4402-aae5-bd2fcbe19f40","active":true,"displayName":"KOTestGroup23","members":[{"value":"e07986cd-6ce9-47f0-8ba7-c7d316311653"},{"value": │
│ "6c320c45-0033-4d10-bdda-3c711d720025"}],"meta":{"resourceType":"Group"}}}
```

In this code example, the displayName has changed from "KOTestGroup23", to "KOTestGroup Test". The members are unchanged. But modifyGroup never gets the display name in the list of attributes to process. Looking through the code it looks like the PUT group code is just looking for members to add or remove, but nothing else.

The PATCH group code does handle updating the display name.

Okta uses PUT, but Azure uses PATCH.

**Comment by jelhub** [comment]
PUT now fixed in v4.2.8
Also did some cosmetics related to #94, you may retest this one.
FYI, Travis build test fails for v4.2.8. Seems to be a Travis problem.

**Comment by osbornk** [comment]
Thanks. Everything looks good to me.
